app/javascript/packs/find-out.js

```diff
@@ -1,207 +1,186 @@
-      resultsDiv.append(announcementCard)
+      resultsDiv.append(
+        new Announcement(tagging.data.attributes.tagged_item).announcementCardHtml(),
+      );
     } else if (tagging.data.attributes.report_id) {
-      const report = new Report(tagging.data.attributes.tagged_item)
-      const reportCard = report.reportCardHtml()
-      resultsDiv.append(reportCard)
+      resultsDiv.append(
+        new Report(tagging.data.attributes.tagged_item).reportCardHtml(),
+      );
     } else if (tagging.data.attributes.event_id) {
-      const event = new Event(tagging.data.attributes.tagged_item)
-      const eventCard = event.eventCardHtml()
-      resultsDiv.append(eventCard)
+      resultsDiv.append(
+        new Event(tagging.data.attributes.tagged_item).eventCardHtml(),
+      );
     }
-  })
-}
+  });
+};
 
 const loadInitialCards = (taggings, resultsDiv, dropdownsObject) => {
   if (taggings.length === 0) {
-    $('.results').html('<div class="message--none">There are currently no results for the selected filters.</div>')
-    hideLoadMoreButton()
+    $('.results').html('<div class="message--none">There are currently no results for the selected filters.</div>');
+    $('#load-more').hide();
   } else if (dropdownsObject.content_type === 'events') {
-    createCards(taggings.slice(0, 8), resultsDiv)
+    createCards(taggings.slice(0, 8), resultsDiv);
     if (taggings.length > 8) {
-      showLoadMoreButton(taggings.slice(8, taggings.length - 1), resultsDiv)
+      showLoadMoreButton(taggings.slice(8, taggings.length - 1), resultsDiv);
     } else {
-      hideLoadMoreButton()
+      $('#load-more').hide();
     }
   } else {
-    createCards(taggings.slice(0, 9), resultsDiv)
+    createCards(taggings.slice(0, 9), resultsDiv);
     if (taggings.length > 9) {
-      showLoadMoreButton(taggings.slice(9, taggings.length - 1), resultsDiv)
+      showLoadMoreButton(taggings.slice(9, taggings.length - 1), resultsDiv);
     } else {
-      hideLoadMoreButton()
+      $('#load-more').hide();
     }
   }
-}
+};
 
 const loadNextThreeEvents = (events, resultsDiv) => {
   const nextThree = events.map(event => {
-    return new Event(event)
-  })
-  const nextThreeEventsHtml = Event.nextThree(nextThree)
-  resultsDiv.prepend(nextThreeEventsHtml)
-}
+    return new Event(event);
+  });
+  const nextThreeEventsHtml = Event.nextThree(nextThree);
+  resultsDiv.prepend(nextThreeEventsHtml);
+};
 
 const loadRemainingCards = (taggings, resultsDiv) => {
-  createCards(taggings, resultsDiv)
-  hideLoadMoreButton()
-}
-
-const hideLoadMoreButton = () => {
-  $('#load-more').hide()
-}
+  createCards(taggings, resultsDiv);
+  $('#load-more').hide();
+};
 
 const showLoadMoreButton = (remainingCards, resultsDiv) => {
-  $('#load-more').show()
-  $('#load-more').unbind('click')
+  $('#load-more').show();
+  $('#load-more').unbind('click');
   $('#load-more').bind('click', (event) => {
-    event.preventDefault()
```

Comment:
When you click Load More in production, you stay at the same exact URL (`/find-out`) and at the same point on the page. When you click it in on this current branch, you go back up to the top and the URL is updated to `/find-out#`. It looks like it's related to the nixed `event.preventDefault()`